lib/simple_health_check/http_endpoint_check.rb

```
@@ -14,7 +14,9 @@ def call(response:)
       http.use_ssl = true if @url.include?('https')
       request = Net::HTTP::Get.new(uri)
       resp = http.request(request)
-      (resp.read_body['status'] && :ok) || false
+      body = JSON.parse(resp.read_body) rescue {}
```

should we store/detect this `JSON.parse` failure, instead of just returning `{}`?   Should the types of failures from this line being tracked or logged be helpful in some cases?   I'm not sure if it is, so just wondering what your opinion on that is.

---

I am not sure if its needed. Still, I have updated the code to log the error. Please have a look at it.